Jenkinsfile

```diff
@@ -3,10 +3,14 @@
 import org.stan.Utils
 
 def utils = new org.stan.Utils()
-def branch = ''
 
 pipeline {
     agent { label 'gelman-group-mac' }
+    environment {
+        cmdstan_pr = ""
+        stan_pr = ""
+        math_pr = ""
```

Comment:
I would just write a function that returns the thing you want and call it on each of these parameters:
```
def branchOrPR(pr) {
  if (pr == 'downstream_tests") return "develop"
  if (pr == "downstream_hotfix") return "master"
  pr
}
```
and then use it like so:

```
sh ...
                        cmdstan_pr=${branchOrPR(cmdstan_pr)} ...
                        bash compare-git-hashes.sh develop \$cmdstan_hash stat_comp_benchmarks ${branchOrPR(stan_pr)} ${branchOrPR(math_pr)} <don't need the false anymore>
```